### PR TITLE
Fix assets design in session detail

### LINKED ISSUE #161: add Icon and fix design assets section in session detail screen
**Idea Description**

-  when items tapped , open url 

**Reference images and links**

Design Kit
https://www.figma.com/file/XsVzpDZSkEQANgCZLRTDDy/DroidKaigi-2022-Material-3-Design-Kit?node-id=11%3A1833
App Design
https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=0%3A1

### PR DESCRIPTION
**Pull request by blackbracken**
## Issue
- close #161 .

## Overview (Required)
- Add icons
- Open the link on clicking assets if it exists

## Links
- [App design](https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=0%3A1)

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDetail.kt [hunk 1]
```diff
@@ -369,28 +376,70 @@ fun SessionDetailAssets(
...
 }
 
-@Preview
+@Preview(showBackground = true)
 @Composable
 fun PreviewSessionDetailScreen() {
-    SessionDetailScreen(
-        uiModel = SessionDetailUiModel(
-            Loaded(TimetableItemWithFavorite.fake())
+    KaigiTheme {
```
Comment (takahirom) [inline feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDetail.kt:438]: 👍

**Review by takahirom — APPROVED** [review]
Looks good! Thank you!

**Comment by blackbracken** [comment]
Thanks!

In addition, I have a minor (and unrelated) question. I was unable to get Compose Preview to work on a device or emulator as is in my Android Studio due to permission.
I solved it by adding android.permission.INTERNET to AndroidManifest.xml directly under the module (= `:feature-sessions`) temporarily.
Is this a problem only in my environment?

**Comment by takahirom** [comment]
Let me check it

**Comment by takahirom** [comment]
I can see them :eyes:
image

**Comment by takahirom** [comment]
Thanks!

**Comment by blackbracken** [comment]
Sorry for my lack of explanation, I mean Compose Deploy Preview like:
image

Either way, I don't see a problem with this implementation 😅.

**Pull request merged**